### PR TITLE
MGDOBR-1000: Bridge/Processor: Expose modifiedAt to the UI

### PR DESCRIPTION
**Pull request by manstis**
[https://issues.redhat.com/browse/MGDOBR-1000](https://issues.redhat.com/browse/MGDOBR-1000)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] The layers in the `kustomize` folder have been updated accordingly.
- [x] All new functionality is tested
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

**Review by lucamolteni — APPROVED** [review]

**Review by kelvah — APPROVED** [review]

**Review by vpellegrino — APPROVED** [review]
LGTM

**Review by robpblake — APPROVED** [review]

**Comment by r00ta** [comment]
Can't wait for MGDOBR-1000 to get in

File: manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java [hunk 1]
```diff
@@ -357,6 +362,28 @@ void testUpdateBridgeErrorHandlerWithNewErrorHandler() {
...
 
+    @Test
+    void testToResponse() {
+        Bridge bridge = createPersistBridge(READY);
+        bridge.setModifiedAt(ZonedDateTime.now(ZoneOffset.UTC));
+
+        BridgeResponse response = bridgesService.toResponse(bridge);
+
+        assertThat(response.getId()).isEqualTo(bridge.getId());
+        assertThat(response.getId()).isEqualTo(bridge.getId());
```
Comment (jomarko) [inline manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java:373]: Please fix this duplicity

File: manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java [hunk 2]
```diff
@@ -357,6 +362,28 @@ void testUpdateBridgeErrorHandlerWithNewErrorHandler() {
...
+        assertThat(response.getName()).isEqualTo(bridge.getName());
+        assertThat(response.getEndpoint()).isEqualTo(bridge.getEndpoint());
+        assertThat(response.getSubmittedAt()).isEqualTo(bridge.getSubmittedAt());
+        assertThat(response.getPublishedAt()).isEqualTo(bridge.getPublishedAt());
+        assertThat(response.getModifiedAt()).isEqualTo(bridge.getModifiedAt());
+        assertThat(response.getStatus()).isEqualTo(bridge.getStatus());
+        assertThat(response.getHref()).contains(bridge.getId());
```
Comment (jomarko) [inline manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java:380]: is this intentional?

Comment (jomarko) [reply manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java:380]: i do not see we set `href` as `id` in
```java
    protected Bridge createPersistBridge(String bridgeId, ManagedResourceStatus status) {
        Bridge b = Fixtures.createBridge();
        b.setId(bridgeId);
        b.setStatus(status);
        bridgeDAO.persist(b);
        return b;
    }
```

**Review by jomarko — COMMENTED** [review]
Thank you for a PR @manstis

File: manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java (hunk 1 above)
Comment (manstis) [reply manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java:373]: Done.

File: manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java (hunk 2 above)
Comment (manstis) [reply manager/src/test/java/com/redhat/service/smartevents/manager/BridgesServiceTest.java:380]: Yes this is intentional. The `href` is set [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/f58125aec93e3041e44d504c25fb58b6e0e134e8/manager/src/main/java/com/redhat/service/smartevents/manager/BridgesServiceImpl.java#L293) and **contains** the `bridgeid`.

**Review by jomarko — APPROVED** [review]
Thank you @manstis .

**Pull request merged**